stories/webapp/accountdashboard/WordPressIntegrationStory.js

```diff
@@ -0,0 +1,105 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { withModalTrigger, Modal, Button, InputWithButton, Icon } from 'lib';
+
+const stories = storiesOf('Web app', module);
+
+const ModalTrigger = withModalTrigger({
+  children: <Icon name="integration" />,
+  types: ['icon-only']
+});
+
+stories.add('WordPressIntegrationStory', () => {
+  const modalSubHeader = (
+    <Fragment>
+      <Icon name="integration" className="m-quarter mr-half" />
+      Wordpress v5.4
+      <span className="text-xs text-green-primary bg-green-90 px-quarter py-tenth inline-flex rounded-sm font-semi-bold mx-quarter">
+        CONNECTED
+      </span>
+      <span className="text-neutral-primary absolute right-0 mr-base">
+        Expires on 30/04/2020
+      </span>
+    </Fragment>
+  );
+
+  return (
+    <ModalTrigger>
+      <Modal.Container>
+        <Modal.Header text={modalSubHeader}>WordPress demo</Modal.Header>
+        <Modal.Body className="has-columns">
```

Comment:
💚 I think when this goes into the App, it would be nice to have the `<Modal.Body/>` and it's contents in a separate component. 

This file is a little long as is (but is ok as it's a story). Once you shift to the App and add state in etc. it's going to get lengthy.